### DISCUSSION TITLE
Welcome to thorium-libjxl Discussions!

### DISCUSSION THREAD

**Discussion by Alex313031** [Alex313031/thorium-libjxl#2] [Announcements]
Heller, fellow Chromium enthusiasts, photographers, Thorium users, JPEG-XL Developers, and project maintainers!
(BTW heller is from Madea) <img src="https://media.tenor.com/p3xFNW1o-Z4AAAAC/this-is-a-hello-madea.gif" width="128">

This repo will be used as a submodule for the main Thorium repo, to reintroduce libjpeg-xl support post chromium M109.

Please do not make any pushes to this repo without my approval. I am not much of a control freak in real life, but when it comes to my repos, I want to know EVERYTHING that goes in or out of them.

I am very excited to have a googler and libjxl maintainers @jonsneyers @ms271 interested in this.

This repo will host .patch files for SVN and GIT, HOWEVER (and I seem to be weird about this, noone else seems to do this or like it this way), the way it will actually be used is a full directory tree mirroring the chromium one. This is how thorium is done too. I made a bookmarks file just like in the main thorium repo, which opens the chromium code search for each file. This is how I manually rebase, by checking changes and copy/pasting or deleting lines as necessary when a file changes from branch to branch.

Google has been kind of an asshole lately, removing API keys, buying Neverware, killing stadia, and removing jpegxl, as well as webvr, and other useful web standards (or burgeoning ones).

Once this sub project is complete, thorium will have the most media support of *ANY* browser. It will support HEVC, AC3, MPEG4, and JPEG-XL. I am quite proud of this, but I could not complete this without the help of you guys, and @midzer & @gz83.

Please use this discussions as a place for literally anything, it doesn't even have to be about jpegxl.

**Comment by gz83** [comment]
I think we may have temporarily finished our work in this area, since we have regained support for jxl, although it is now based on Rust.
